gulpfile.js

```diff
@@ -32,7 +32,8 @@ gulp.task("lint", () => {
                 "eqeqeq": 1,
                 "camelcase": 1,
                 "comma-dangle": 2,
-                "quotes": [1, "double"]
+                "quotes": [1, "double"],
+                "keyword-spacing": [1, { "after": true }]
```

Comment:
Is this what you meant? Btw, i'm a fan of [comma-dangle](http://eslint.org/docs/rules/comma-dangle) which makes for cleaner diffs (and simplifies adding and removing items). For instance, this diff would have been easier to read.